static/app/components/commitLink.tsx

```diff
@@ -76,7 +76,7 @@ function CommitLink({inline, commitId, repository}: Props) {
       {shortId}
     </Button>
   ) : (
-    <ExternalLink className="inline-commit" href={commitUrl}>
```

Comment:
I found one hit in `static/less/group-detail.less`

Reply:
oh nice, it doesn't really do anything important 😏

Reply:
Fixed, this should be good to go now